test/modules/TestResultCoordinator/TestResultCoordinatorClient.cs

```diff
@@ -0,0 +1,280 @@
+﻿//----------------------
+// <auto-generated>
+//     Generated using the NSwag toolchain v13.1.6.0 (NJsonSchema v10.0.28.0 (Newtonsoft.Json v11.0.0.0)) (http://NSwag.org)
+// </auto-generated>
+//----------------------
+
+#pragma warning disable 108 // Disable "CS0108 '{derivedDto}.ToJson()' hides inherited member '{dtoBase}.ToJson()'. Use the new keyword if hiding was intended."
```

Comment:
This file need to put in Module.Util project in order to share with other module project to report test result.